baremaps-core/src/test/java/org/apache/baremaps/workflow/tasks/DownloadUrlTest.java

```diff
@@ -34,6 +36,32 @@ void execute() throws Exception {
     assertTrue(Files.readString(file.toPath()).contains("Baremaps"));
   }
 
+  @Test
+  @Tag("integration")
```

Comment:
I would ignore (@Ignore) this test for now, as it relies on a third party url. The integration tests are executed by github actions.

Reply:
addressed in [6244dc9](https://github.com/apache/incubator-baremaps/pull/698/commits/6244dc9007ff9395db9a8803eca12306acf48a33) with Junit5 @Disabled